core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerEventForwarder.java

```diff
@@ -1,122 +0,0 @@
-package com.novoda.noplayer.exoplayer;
-
-import android.view.Surface;
-
-import com.google.android.exoplayer2.ExoPlaybackException;
-import com.google.android.exoplayer2.Format;
-import com.google.android.exoplayer2.PlaybackParameters;
-import com.google.android.exoplayer2.Timeline;
-import com.google.android.exoplayer2.decoder.DecoderCounters;
-import com.google.android.exoplayer2.source.TrackGroupArray;
-import com.google.android.exoplayer2.trackselection.TrackSelectionArray;
-import com.google.android.exoplayer2.upstream.DataSpec;
-
-import java.io.IOException;
-
-abstract class ExoPlayerEventForwarder implements ExoPlayerForwarder {
```

Comment:
🤢  we implemented all events on this one forwarder. We now have dedicated forwarders and a generic `forwarder` that has the specific forwarders as collaborators.